cdap-data-fabric/src/test/java/io/cdap/cdap/spi/data/nosql/NoSqlStructuredTableTest.java

```diff
@@ -102,6 +103,18 @@ public void testScannerIteratorMulti() throws Exception {
     testScannerIterator(10);
   }
 
+  @Override
+  @Ignore
+  public void testIndexScanWithRange() throws Exception {
+    // Should be removed after https://cdap.atlassian.net/browse/CDAP-19564
```

Comment:
we can remove this from here?

Reply:
If I remove it here, the tests will break

Reply:
The tests I added in this PR is in abstract class `StructuredTableTest`, regardless the implementation (will be run by three table implementations)